More tweaks to wasm landing page (#8349)
flutter/website: src/development/platform-integration/web/wasm.md

short-title: Wasm
---

_WebAssembly support for Dart and Flutter is under active development. We hope
to have the feature ready to try out later in 2023._

The Flutter and Dart teams are excited to add
[WebAssembly](https://webassembly.org/) as a compilation target when building
applications for the web.

Compiling Dart to this new target requires
[WebAssembly Garbage Collection (WasmGC)](https://github.com/WebAssembly/gc/tree/main/proposals/gc),
an upcoming addition to the WebAssembly standard. The Chrome team is actively
[working on WasmGC](https://chromestatus.com/feature/6062715726462976).

> In the mean time, checkout Flutter's
> [existing support for the web](https://flutter.dev/multi-platform/web). We are
> optimistic that existing Flutter web applications will have to do little or no
> work to support WebAssembly once work is complete.

Reviewer: ```suggestion
In the mean time, check out Flutter's
[existing support for the web]({{site.main-url}}/multi-platform/web). We are
optimistic that existing Flutter web applications will have to do little or no
work to support WebAssembly once work is complete.
```

> You may also be interested in this talk by folks on our team:
> [Flutter, Dart, and WasmGC: A new model for web applications](https://wasmio.tech/sessions/flutter-dart-and-wasm-gc-a-new-model-for-web-applications/).

Reviewer: ```suggestion
You might also be interested in this talk by folks on our team:
[Flutter, Dart, and WasmGC: A new model for web applications]
(https://wasmio.tech/sessions/flutter-dart-and-wasm-gc-a-new-model-for-web-applications/).
```

> Check out [flutter.dev/wasm]({{site.main-url}}/wasm) for the latest updates
> on this effort.

Reviewer: I'd prefer a name for this page rather than a url.

```suggestion
For the latest updates on this effort, 
check out [flutter.dev/wasm]({{site.main-url}}/wasm).
```
